File: RentABikeWTR_v1_7.Model/XRezervacije.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RentABikeWTR_v1_7.Model
{
    public class XRezervacije
    {
        public int RezervacijaId { get; set; }
        public DateTime Datum { get; set; }
        public decimal? CijenaUsluge { get; set; }
        public int? KupacID { get; set; }
        public string? KorisnickoIme { get; set; }
        public int? BiciklID { get; set; }
        public string? NazivBicikla {  get; set; }
        public int? TuristRutaID { get; set; }
        public string? Naziv { get; set; }


        
        
    }
}

File: rentabikewtr_desktop/lib/model/xRezervacije.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'xRezervacije.g.dart';

@JsonSerializable()
class XRezervacije {
  int? rezervacijaId;
  DateTime? datum;
  num? cijenaUsluge;
  int? kupacID;
  String? korisnickoIme;
  int? biciklID;
  String? nazivBicikla;
  int? turistRutaID;
  String? naziv;

  // int? turistickiVodicID;
  // int? turistRutaID;

  XRezervacije() {}

  factory XRezervacije.fromJson(Map<String, dynamic> json) =>
      _$XRezervacijeFromJson(json);

  /// Connect the generated [_$XRezervacijeToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$XRezervacijeToJson(this);
}
